newrelic.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['SDC-reviews-service'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: 'd35f1e311eac77c351894b508e5db9fe8499NRAL',

```

Comment:
I wouldn't put my license key on a committed file. Committing this file exposes it. Maybe use environment variables or gitignore this file. I also suggest rebasing and removing this commit from your history after making that change, so that your key is removed from your commit history.